src/wasm_runtime_standalone.js

```diff
@@ -19,13 +19,13 @@ if( typeof Rust === "undefined" ) {
         if( typeof window === "undefined" && typeof process === "object" ) {
             var fs = require( "fs" );
             var path = require( "path" );
-            var wasm_path = path.join( __dirname, "{{{wasm_filename}}}" );
+            var wasm_path = path.join( __dirname, "/{{{wasm_filename}}}" );
```

Comment:
I haven't tested the additional `/` here, but I assume that the `path.join()` function would produce the same output regardless of the `/` being there or not.
Let me know if you would like me to remove this change.